native/src/components/__tests__/NativeHtml.spec.tsx

```diff
@@ -20,11 +18,7 @@ jest.mock('react-redux', () => ({
 jest.mock('@react-navigation/native', () => ({
   useNavigation: () => mockNavigation
 }))
-
-jest.mock('styled-components', () => ({
-  ...jest.requireActual('styled-components'),
-  useTheme: () => mockTheme
-}))
+jest.mock('styled-components')
```

Comment:
Why do we need to mock styled-components here, but do not need to mock it in e.g. Header.spec.ts? Both use `useTheme`?

Reply:
Because in `Header.tsx` the theme is only passed to styled components (which is redundant) and to a component which is mocked. Therefore it does not matter that is undefined.